src/main.rs

```diff
@@ -16,24 +16,21 @@
  * along with Compiler-Bot.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-#![allow(internal_features)]
-#![feature(prelude_future)]
-#![feature(prelude_import)]
-
 use std::env;
 
+use dotenv::dotenv;
```

Comment:
`dotenv` is unmaintained ([RUSTSEC-2021-0141](https://rustsec.org/advisories/RUSTSEC-2021-0141.html)). Might be better to use [`dotenvy`](https://crates.io/crates/dotenvy) instead.